packages/base-styles/_colors.native.scss

```
@@ -105,7 +105,7 @@ $gray-text-min: darken($gray, 18%); //#537994
 
 // Shades of gray
 $gray-lighten-10: lighten($gray, 10%); // #a8bece
-$gray-lighten-20: lighten($gray, 20%); // #c8d7e1
+$studio-gray-10: #c3c4c7;
```

I think we should also add design token type variable. 

So something like:
```suggestion
$studio-gray-10: #c3c4c7;

$block-line-color: $studio-gray-10;
$icon-uploading-fill: $studio-gray-10;
```

---

@enejb @Guarani Any chance we can skip the Studio grays and translate the old grays to our neutral grays (in the above case, `gray-lighten-20` would become our "tertiary" neutral)?

---

To clarify, we'd keep the base declarations for the studio grays in the project, but for most elements, the neutral grays would be preferred.

---

SGTM 🙂